bot/src/discord/buttons/PlaySoundButton.ts

```diff
@@ -0,0 +1,111 @@
+import { ButtonInteraction, PermissionFlagsBits } from 'discord.js';
+import { prismaClient } from '../../bot';
+import { DEFAULT_AIRHORN_MAX_QUEUE_ITEMS, enqueSound, getTotalItemsInGuildQueue } from '../../utils/AirhornAudio';
+import { DiscordButton } from '../types/DiscordButton';
+
+export class PlaySoundButton extends DiscordButton {
+  constructor() {
+    super('play_sound', {
+      version: 1,
+    });
+  }
+
+  async handle(buttonInteraction: ButtonInteraction): Promise<unknown> {
```

Comment:
I am skeptical of the `unknown` here :) Do we really not know what it returns?

Reply:
The return value was ignored, but it let me do things like:
```ts
return commandInteraction.reply({content: "test"});
```
instead of 
```ts
await commandInteraction.reply({content: "test"});
return;
```
I've looked a little bit more into this, and it helps with stack traces if you await before returning but there are some other changes done. Up to you on what you think I should do, but I've gone ahead and switched it to the second one since it was a little bit inconsistent across all of the buttons and commands.
 - https://eslint.org/docs/latest/rules/no-return-await
 - https://jakearchibald.com/2017/await-vs-return-vs-return-await/